examples/src/main/java/com/mapbox/navigation/examples/core/MapboxNavigationActivity.kt

```diff
@@ -493,9 +503,25 @@ class MapboxNavigationActivity : AppCompatActivity() {
         )
     }
 
-    private fun setRouteAndStartNavigation(route: List<NavigationRoute>) {
+
+
+    private fun setRoutePreview(route: List<NavigationRoute>) {
         // set route
-        mapboxNavigation.setNavigationRoutes(route)
+        mapboxNavigation.previewNavigationRoutes(route)
+        binding.navigate.visibility = VISIBLE
+        binding.navigate.setOnClickListener {
+            setRoute()
+            binding.navigate.visibility = INVISIBLE
+        }
+
+        // move the camera to overview when new route is available
+        navigationCamera.requestNavigationCameraToOverview()
+    }
+
+    private fun setRoute() {
+        // set route
+        mapboxNavigation.setNavigationRoutes(mapboxNavigation.getNavigationRoutes())
```

Comment:
What's the expected impact of this invocation?
UPD: Ah, I see, it was updated when you set preview routes. OK, it looks super weird... I may be changing my mind whether `getNavigationRoutes` should return previewed routes. What about a separate method `getPreviewedNavigationRoutes`? I'm still worried about `RoutesObserver` tho. It kinda implies (and specifies in the docs) that these are navigation routes.

Reply:
I agree. It looks strange. Added suggested `getPreviewedNavigationRoutes`